### PR TITLE
recommend using rollup and changesets for npm modules

### PR DESCRIPTION
**Pull request by sndrs**
update packages recommendations to use rollup and changesets

### CODE REVIEW

**Review by joecowton1 — APPROVED** [review]
Looks good!

File: npm-packages.md
Comment (tjsilver) [inline npm-packages.md:22]: you -> your?

**Review by SiAdcock — COMMENTED** [review]
Love it ❤️ 

There's a couple of recommendations from the [security review of changesets](https://docs.google.com/document/d/19NBdyxKJGSuo8VCwdq7KYxRwm1FaJXoR0cqT0i5jhfU/edit#heading=h.royzz2hqw13g) that it might be appropriate to mention here:

- To minimise the risk of exposing private information the changesets-bot should not be installed on private repositories. 
- The changesets-bot application has permissions to merge PRs through the ‘pull request write permission”. To mitigate this all repositories where the changesets-bot is installed should have branch protection enabled with “Require approvals” from Guardian staff before merging. 

What do you think?

**Comment by sndrs** [comment]
> To minimise the risk of exposing private information the changesets-bot should not be installed on private repositories

a github admin would need to install it, so i think this is ok

the value of branch protection kind of depends on the repo in question i think?

**Pull request merged**
